Sources/Defaults/Observation.swift

```diff
@@ -200,11 +211,108 @@ extension Defaults {
 			else {
 				return
 			}
+			
+			
+			let key = preventPropagationThreadDictKey
+			let updatingValuesFlag = (Thread.current.threadDictionary[key] as? Bool) ?? false
```

Comment:
```suggestion
			let updatingValuesFlag = Thread.current.threadDictionary[key, default: false]
```

Reply:
This is `NSMutableDictioary`, so it doesn't support that.